test/e2e/performance_test.go

```diff
@@ -60,64 +69,59 @@ func BenchmarkHTTPRequest(b *testing.B) {
 		{100000, 15000},
 	} {
 		b.Run(fmt.Sprintf("Request:%d,PolicyRules:%d", scale.requests, scale.policyRules), func(b *testing.B) {
-			withPerformanceTestSetup(func(data *TestData) { httpRequest(scale.requests, scale.policyRules, data, b) }, b)
+			withPerfTestSetup(func(data *TestData) { httpRequest(scale.requests, scale.policyRules, data, b) }, b)
 		})
 	}
 }
 
 func BenchmarkRealizeNetworkPolicy(b *testing.B) {
 	for _, policyRules := range []int{5000, 10000, 15000} {
 		b.Run(fmt.Sprintf("RealizeNetworkPolicy%d", policyRules), func(b *testing.B) {
-			withPerformanceTestSetup(func(data *TestData) { networkPolicyRealize(policyRules, data, b) }, b)
+			withPerfTestSetup(func(data *TestData) { networkPolicyRealize(policyRules, data, b) }, b)
 		})
 	}
 }
 
 func BenchmarkCustomizeHTTPRequest(b *testing.B) {
 	if *customizeRequests == 0 {
-		b.Skip("The value of performance.http.requests=0, skipped")
+		b.Skip("The value of perf.http.requests=0, skipped")
 	}
-	withPerformanceTestSetup(func(data *TestData) { httpRequest(*customizeRequests, *customizePolicyRules, data, b) }, b)
+	withPerfTestSetup(func(data *TestData) { httpRequest(*customizeRequests, *customizePolicyRules, data, b) }, b)
 }
 
 func BenchmarkCustomizeRealizeNetworkPolicy(b *testing.B) {
 	if *customizePolicyRules == 0 {
-		b.Skip("The value of performance.http.policy_rules=0, skipped")
+		b.Skip("The value of perf.http.policy_rules=0, skipped")
 	}
-	withPerformanceTestSetup(func(data *TestData) { networkPolicyRealize(*customizePolicyRules, data, b) }, b)
+	withPerfTestSetup(func(data *TestData) { networkPolicyRealize(*customizePolicyRules, data, b) }, b)
 }
 
 func randCidr(rndSrc rand.Source) string {
 	return fmt.Sprintf("%d.%d.%d.%d/32", rndSrc.Uint64()%255+1, rndSrc.Uint64()%255+1, rndSrc.Uint64()%255+1, rndSrc.Uint64()%255+1)
 }
 
-// createPerformanceTestPodSpec creates the Pod specification for the performance test.
+// createPerfTestPodDefinition creates the Pod specification for the perf test.
 // The Pod will be scheduled on the master Node.
-func createPerformanceTestPodSpec(name, containerName, image string) *v1.Pod {
-	podSpec := v1.PodSpec{
-		Containers: []v1.Container{
+func createPerfTestPodDefinition(name, containerName, image string) *corev1.Pod {
```

Comment:
why did you replace Spec with Definition?

Reply:
The result is not a `PodSpec` but a `Pod`, and I think `*Spec` may indicate a `PodSpec` return.

Reply:
lol that's true, I read too fast...